recipes/superlu_dist/meta.yaml

```diff
@@ -0,0 +1,54 @@
+{% set name = "superlu_dist" %}
+{% set version = "5.1.3" %}
+{% set build = 0 %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn: superlu_dist_{{ version }}.tar.gz
+  url: http://crd-legacy.lbl.gov/~xiaoye/SuperLU/superlu_dist_{{ version }}.tar.gz
+  sha256: 557bc5ef7bbfbf2f8bab65d454f89ecb99d9565bba949325739d59b63f580323
+
+build:
+  number: {{ build }}
+  binary_relocation: true
+  skip: true  # [win]
+  string: "{{ PKG_HASH }}_{{ build }}"
+
+requirements:
+  build:
+    - {{ compiler('c') }}
+    - {{ compiler('cxx') }}
+    - {{ compiler('fortran') }}
+    - cmake >=2.8.12
+    - gcc  # [osx]
+    - openblas  # [unix]
+    - parmetis 4.0.3*
+    - metis 5.1.0*
+  host:
+    - parmetis 4.0.3*
+    - metis 5.1.0*
+    - openmpi
+  run:
+    - libgcc  # [osx]
```

Comment:
libgcc should normally be a strong run_exports of gcc, so I don't think you need to mention it.